mlir/include/mlir/Dialect/GPU/IR/GPUOps.td

```diff
@@ -588,7 +588,7 @@ def GPU_LaunchOp : GPU_Op<"launch", [
                Index:$gridSizeX, Index:$gridSizeY, Index:$gridSizeZ,
                Index:$blockSizeX, Index:$blockSizeY, Index:$blockSizeZ,
                Optional<I32>:$dynamicSharedMemorySize,
-               OptionalAttr<SI32Attr>:$dynamicSharedMemorySizeConstant)>,
+               OptionalAttr<I32Attr>:$dynamicSharedMemorySizeConstant)>,
```

Comment:
Why this change?

Reply:
I32Attr becomes `uint32`, SI32Attr becomes `int32`. `uint32` larger range.